### PR TITLE
Node/CCQ: Listening tool

### PR DESCRIPTION
**Pull request by bruce-riley**
This is a tool to verify CCQ communications with the guardian.

Please see the comments at the top of `node/hack/query/ccqlistener/ccqlistener.go` for how to use it.

### CODE REVIEW

File: node/hack/query/ccqlistener/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# syntax=docker.io/docker/dockerfile:1.3@sha256:42399d4635eddd7a9b8a24be879d2f9a930d0ed040a61324cfdf59ef1357b3b2
+FROM --platform=linux/amd64 docker.io/golang:1.20.10-bullseye@sha256:082569b3303b164cc4a7c88ac59b19b69c1a5d662041ac0dca046ac239632442 AS build
+# libwasmvm.so is not compatible with arm
+
+# Support additional root CAs
```
Comment (evan-gray) [inline node/hack/query/ccqlistener/Dockerfile:5]: I don't think these `additional root CAs` things are included anymore

**Review by evan-gray — APPROVED** [review]

**Review by evan-gray — COMMENTED** [review]
also, running docker build per the comment gave me
```
ERROR: failed to solve: process "/bin/sh -c export CGO_ENABLED=1 &&   cd node/hack/query/ccqlistener &&   go build ${GO_BUILD_ARGS} -gcflags=\"all=-N -l\" --ldflags '-extldflags \"-Wl,--allow-multiple-definition\" -X \"github.com/certusone/wormhole/node/cmd/guardiand.Build=dev\"' -mod=readonly -o /ccqlistener mainnet.go &&   go get github.com/CosmWasm/wasmvm@v1.1.1 &&   cp /go/pkg/mod/github.com/!cosm!wasm/wasmvm@v1.1.1/internal/api/libwasmvm.x86_64.so /usr/lib/" did not complete successfully: exit code: 1
```

is `mainnet.go` maybe not the correct file anymore?

File: node/hack/query/ccqlistener/Dockerfile (hunk 1 above)
Comment (bruce-riley) [reply node/hack/query/ccqlistener/Dockerfile]: Remove this.

**Comment by bruce-riley** [comment]
Fixed the docker file so the image builds now.

**Review by evan-gray — APPROVED** [review]
builds now, thanks 🙂

**Review by panoel — APPROVED** [review]

**Pull request merged**